core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-theme-selector/template-builder-theme-selector.component.scss

```diff
@@ -0,0 +1,65 @@
+@use "variables" as *;
+
+:host {
+    width: 100%;
+    display: block;
+}
+
+.theme-search-box {
+    position: relative;
+
+    input {
+        font-size: $font-size-lg;
+        border: 1px solid $color-palette-gray-500;
+        padding: $spacing-3;
+        border-radius: $spacing-1;
+    }
+}
+
+.theme__header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.theme-item {
+    align-items: center;
+    background: $color-palette-gray-200;
+    border-radius: 4px;
```

Comment:
Use rem values, check if there is a scss variable that has this value

Reply:
We don't have that var yet but it should be like `$border-XXX` search there might be some.